File: app/src/main/java/com/witype/Dragger/mvp/model/ApiModel.java
package com.witype.Dragger.mvp.model;

import com.witype.Dragger.entity.MobileDateUsageEntity;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApiModel {

    @GET(Api.datastoreSearch)
    Observable<MobileDateUsageEntity> getMobileDataUsage(@Query("resource_id") String resourceId, @Query("limit") int limit);
}

File: kotlindemo/src/main/java/com/witype/kotlindemo/mvp/model/ApiModel.kt
package com.witype.kotlindemo.mvp.model

import com.witype.kotlindemo.entity.MobileDateUsageEntity
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Query

interface ApiModel {

    @GET(Api.datastoreSearch)
    fun getMobileDataUsage(@Query("resource_id") resourceId: String, @Query("limit") limit: Int): Observable<MobileDateUsageEntity>

}